### DISCUSSION TITLE
Table in Grid Item Falling to next line

### DISCUSSION THREAD

**Discussion by MoeCarrim** [KevinVandy/material-react-table#203] [Q&A]
Good day

I have a grid container with two grid items inside. I want to display the Table in one Grid item and a form in the other, and they should display side by side. 

However, The table is overflowing to a new line. 

Is there any way to do this as I cannot find anything in the docs to set the tables overflow properties.

See screenshot for current result.

Thanks
![TableOverflow](https://user-images.githubusercontent.com/25811691/202205091-b34b4aba-5e55-44a0-9898-96353ff8fe00.png)

**Comment by KevinVandy** [comment]
This seems more like a question about how CSS works rather than anything to do with the library. I suggest learning about CSS Flexbox and CSS grid.
